packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -62,28 +65,36 @@ export default declare((api, options) => {
 
           const isCall = t.isCallExpression(node);
           const replaceKey = isCall ? "callee" : "object";
-          const chain = node[replaceKey];
+
+          const chainWithTypes = node[replaceKey];
```

Comment:
nit: I think the naming `chainWithTypes` does not cover the `ParenthesizedExpression` cases so I prefer `chain` instead. Maybe we can reuse `skipTransparentExprWrappers` here:

```js
const chain = skipTransparentExprWrappers(path.get("replaceKey")).node
```

Reply:
We actually need chain still (it's redeclared below), so can't reuse that name. The idea is that further down, we sometimes care about the types, and other times we want to ignore them. I'd be open to an alternative - `chainWithoutWrappers` perhaps?